src/transport_picoquic.cpp

```diff
@@ -623,6 +677,58 @@ PicoQuicTransport::start()
     return cid;
 }
 
+void PicoQuicTransport::PicoQuicLogging()
+{
+    std::array<char, 1024> buffer;
+    ssize_t octets_read;
+    fd_set pipe_fds;
+
+    cantina::LoggerPointer pico_logger =
+        std::make_shared<cantina::Logger>("PQIC", logger);
+
+    while (!stop) {
+        // Prepare file descriptor to read
+        FD_ZERO(&pipe_fds);
+        FD_SET(logging_fds[0], &pipe_fds);
+
+        // Wait until something is ready to read
+        if (select(logging_fds[0] + 1,
+                    &pipe_fds,
+                    nullptr,
+                    nullptr,
+                    nullptr) == -1) {
+            pico_logger->error << "Select for logging failed" << std::flush;
+            // Set picoquic debug to stdout
+            debug_set_stream(stdout);
```

Comment:
Normally it's unsafe to call any picoquic functions outside of the picoquic loop thread from another thread. It's probably okay for now... but.... any reason why this isn't using the pq runner queue?

Reply:
This is in the same thread where `debug_set_stream()` was already called. I didn't introduce anything new here.